benchmarks/bench_isolation_forest.py

```diff
@@ -30,15 +41,14 @@ def print_outlier_ratio(y):
     print("----- Outlier ratio: %.5f" % (np.min(cnt) / len(y)))
 
 
-np.random.seed(1)
+SEED = 1
```

Comment:
I'm more used to see `rng = np.random.RandomState(0)` but I think it doesn't matter.

Reply:
don't use a new name for something we refer to as random_state

use:

random_state = 1

and pass random_state=random_state in function calls.